pkg/network/setup/netconf.go

```diff
@@ -84,18 +91,31 @@ func (c *NetConf) Setup(vmi *v1.VirtualMachineInstance, networks []v1.Network, l
 			return err
 		}
 		ns := c.nsFactory(launcherPid)
-		newConfigState := NewConfigState(configStateCache, ns, launcherPid)
+		newConfigState := NewConfigState(configStateCache, ns)
 		configState = &newConfigState
 		c.configStateMutex.Lock()
 		c.configState[string(vmi.UID)] = configState
 		c.configStateMutex.Unlock()
 	}
 
+	// Absent networks are passed as well since, Absent network with ordinary name has to be plugged
```

Comment:
nit:
``` suggestion
	// Absent networks are passed as well, since Absent networks with ordinal pod iface names (net1, net2,...) should remain plugged because its impossible to associate pod interfaces with ordinal names and VMI spec.
```
We can later on add ref to the hotplug/unplug design doc, the section about the naming issue.

Reply:
Our design doc is pretty outdated, I wouldn't add a link to it from the code.

Reply:
I posted a [PR](https://github.com/kubevirt/community/pull/217) to add more content about the naming change, we can add ref to the naming change section once it's merged.